styles/colors-oklch.css

```diff
@@ -2,86 +2,112 @@
   --white: 100% 0 0;
   --black: 0% 0 0;
 
+  --dark-mod: 5%;
```

Comment:
These might actually be better defined by hand now, but we can use this for the time being and override as required. The "5%" thing was really a hack based on HSL not being perceptually uniform.